lib/app/features/core/providers/app_lifecycle_provider.dart

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'package:flutter/material.dart';
```

Comment:
Providers should ideally not have UI imports. On the other hand, `AppLifecycleState` is just an enum. Let's decide if that's okay for us

Reply:
Discussed.